adapters/jsdom-react/src/index.ts

```diff
@@ -59,7 +59,8 @@ const elementIsFocusable = (el: Element): el is HTMLFocusableElement => {
 		el.tagName === 'INPUT' ||
 		el.tagName === 'SELECT' ||
 		el.tagName === 'TEXTAREA' ||
-		el.tagName === 'BUTTON'
+		el.tagName === 'BUTTON' ||
+		el.tagName === 'svg'
```

Comment:
@ob1  the test passes so I assume it works but I wonder why this one is lower case `svg` and not `SVG`

Reply:
I wondered too, but that's how it is: https://stackoverflow.com/questions/27223756/is-element-tagname-always-uppercase/52035222#52035222